meetings/2022-10-12.md

```diff
@@ -0,0 +1,141 @@
+* If this is your first time, welcome! please introduce yourself.
+
+
+### Scribes
+* Virginia
+
+
+### Introductions
+* name: text
+
+---
+
+
+## Topics
+
+* SC: This meeting follows https://github.com/solid/specification/pull/458 .
+* SC: Today's agenda is intended to help us (re)align a bit as a group.
+* SC: Please note that unless there is a decision to change the meeting datetime, it is always on UTC time (and daylight savings time is not observed as it stands).
+
+
+### Topics of Interest
+
+* SC: The CG weekly meeting will take up topics that are generally of interest to multiple or between contributors.
+* SC: Such as announcements, process, guidelines, feedback, as well as all Work Items.
+* SC: We are mindful of the available time. Deep and detailed discussions are encouraged to continue elsewhere.
+
+
+
+### Panels, Task Forces, and Getting Stuff Done
+
+* SC: We've discussed the state of Panels as well as considerations on forming Task Forces to carry out specific assignments provided that there is commitment, e.g., https://github.com/solid/specification/pull/458 (and links from there).
+* SC: Minor case in point: https://github.com/solid/webid-profile/ initially formed with a group to address a specific need. Similar initiatives could also work out with small or informal charters.
+* SC: Any brief additional feedback on this topic?
+* TH: Very useful to have a weekly meeting. Panels are too frequent, overlapping discussions (authorization, authentication, interop, etc.) which makes it difficult to advance. 
+* SC: This doesn't mean panels are gone. This meeting is to check in as a group. 
+* TBL: If panels are not working, why?
+* TH: Too many to make time for them. Implementing and prepping/attending meetings takes a lot of time, difficult to participate. 
+* eP: Hard to pinpoint which topic belongs to which panel (overlap). 
+* TH: Bundling all meetings in one helps ensure participation for those who have limited time. 
+
+
+
+### Expected Completion of Work Items
+URL: https://solidproject.org/TR/
+
+* SC: The Expected Completion column for the Work Items is updated to reflect information in Charters and based on Current Stage of the Work Item.
+* SC: I'd like to once again ask the contributors of the Work Items to either submit PRs to update the table columns. It should take into account of the goals of the Work Item, as well as test coverage, implementation reports, "adequate implementation experience" and so on.
+* eP: Question on meeting frequency of Editors. Seeing 2 minuted in last 3 months: https://github.com/solid/specification/tree/main/meetings
+* SC: Weekly. Some meetings have been skipped/not minuted for different reasons. 
+* HS: +1 on it sometimes being difficult to work out where to make proposals because of the many overlapping panels. So for example the topic here could be in Authorization, spec, WAC, ACP, ... : https://github.com/solid/specification/issues/325
```

Comment:
```suggestion
* HS: +1 on it sometimes being difficult to work out where to make proposals because of the many overlapping panels. For example, the topic of https://github.com/solid/specification/issues/325 could be in Authorization, spec, WAC, ACP, ... 
```